tests/unit/querybuf.tcl

```diff
@@ -0,0 +1,30 @@
+start_server {tags {"querybuf"}} {
+    test "querybuf will never be resized when querybuf size less than 64k" {
+        set orig_used [s used_memory]
+
+        # memory will increase by more than 32k due to querybuf of new client.
+        set rd [redis_deferring_client]
+        assert_morethan [expr [s used_memory] - $orig_used] 32768
+
+        after 1000
```

Comment:
i'd like to find a way to avoid long sleeps.
if we keep adding long sleeps to the tests, eventually they'll take very long.
instead you should make some loop to wait for your desired effect (and maybe a comment explaining what you do).

what exactly are we waiting for here?
maybe we can skip this part of the test?

Reply:
Does this test also add ```$::accurate```?
This is a test for ```querybuf``` being resized abnormally, which will prevent the bug from occurring again in the future.

Reply:
i think so.. maybe change the text in the doc help for `--accurate` to mention it includes "slow time driven tests" too.
this way we'll spot a regression, it's just that we'll spot it on the next day, and not when the PR that introduces it will be merged.

please also add some comments to both tests as to what you're waiting before each of these sleeps, and why.
and add some comment next to  the `::accurate` check to explain why it's there.